### PR TITLE
LMS-1625 moodle-logstore_xapi: change pagetype to report.

### PR DESCRIPTION
**Pull request by lzabo**
**Description**
Change error report page's type to report.

**Related Issues**
LMS-1625

**PR Type**
Enhancement

### CODE REVIEW

**Review by mlynn-lp — APPROVED** [review]

**Comment by mlynn-lp** [comment]
@lzabo Hi, will this change the page layout to give the report more space at the right hand side? At the moment there is a large area for admin bookmarks.

File: lang/en/logstore_xapi.php [hunk 1]
```diff
@@ -111,6 +111,7 @@
 $string['logstorexapi:managehistoric'] = 'Manage historic data';
 
 // Info strings from xAPI errors
+$string['errorlogpagetitle'] = 'xAPI error log';
```
Comment (gordonmacqueen-lp) [inline lang/en/logstore_xapi.php]: I think we can remove this entry.
We should use `logstorexapierrorlog` instead.

File: report.php [hunk 2]
```diff
@@ -159,6 +163,15 @@
     }
 }
 
+// Define the page layout and header/breadcrumb.
+$PAGE->set_pagelayout('report');
+$PAGE->set_title(get_string('errorlogpagetitle', 'logstore_xapi'));
```
Comment (gordonmacqueen-lp) [inline report.php]: We should use `logstorexapierrorlog` instead

**Review by gordonmacqueen-lp — CHANGES_REQUESTED** [review]
There's a conflict in report.php that needs resolved.

We should merge Stephen's changes for #706 first, and then update this branch as there will be further changes needed.

File: lang/en/logstore_xapi.php (hunk 1 above)
Comment (lzabo) [reply lang/en/logstore_xapi.php]: It was removed.

File: report.php (hunk 2 above)
Comment (lzabo) [reply report.php]: It is changed.

**Review by gordonmacqueen-lp — APPROVED** [review]

File: report.php [hunk 3]
```diff
@@ -138,14 +122,32 @@
 $where = implode(' AND ', $where);
 
 $sql = "SELECT x.id, x.eventname, u.firstname, u.lastname, x.contextid, x.timecreated, $extraselect
-          FROM {$basetable} x
+          FROM {{$basetable}} x
```
Comment (mlynn-lp) [inline report.php:125]: Is this a typo? {{$basetable}}

File: report.php [hunk 4]
```diff
@@ -138,14 +122,32 @@
...
+        }
+    }
+}
+
+// Collect events to create view.
 $results = $DB->get_records_sql($sql, $params, $page*$perpage, $perpage);
 
 $sql = "SELECT COUNT(x.id)
-          FROM {$basetable} x
+          FROM {{$basetable}} x
```
Comment (mlynn-lp) [inline report.php:150]: Is this a typo? {{$basetable}}

File: report.php (hunk 3 above)
Comment (lzabo) [reply report.php:125]: Nope one for concat, one for moodle table handler.

File: report.php (hunk 4 above)
Comment (lzabo) [reply report.php:150]: Nope one for concat, one for moodle table handler.

**Review by gordonmacqueen-lp — APPROVED** [review]

**Pull request merged**